core/domain/feedback_services.py

```diff
@@ -143,26 +191,50 @@ def _get_message_from_model(message_model):
 
 
 def get_messages(exploration_id, thread_id):
+    """Fetch all messages of the given thread indexed by
+        [exploration_id].[thread_id]
+
+    Args:
+        exploration_id: str.
+        thread_id: str.
+
+    Returns:
+        list of FeedbackMessage. Contains all the messages.
+    """
     return [
         _get_message_from_model(m)
         for m in feedback_models.FeedbackMessageModel.get_messages(
             exploration_id, thread_id)]
 
 
 def get_message(exploration_id, thread_id, message_id):
+    """Fetch the message indexed by [exploration_id].[thread_id].[message_id]
+
+    Args:
+        exploration_id: str.
+        thread_id: str.
+        message_id: int.
+    Returns:
+        FeedbackMessage.
+    """
     return _get_message_from_model(
         feedback_models.FeedbackMessageModel.get(
             exploration_id, thread_id, message_id))
 
 
 def get_next_page_of_all_feedback_messages(
         page_size=feconf.FEEDBACK_TAB_PAGE_SIZE, urlsafe_start_cursor=None):
-    """Returns a page of feedback messages in reverse time order.
-
-    The return value is a triple (results, cursor, more) as described in
-    fetch_page() at:
+    """Fetch messages for the next page.
```

Comment:
Fetches a single page from the list of all feedback messages that have been posted to any exploration on the site.